src/Files.App/Data/Items/WindowEx.cs

```diff
@@ -298,16 +297,9 @@ private void WindowEx_Closed(object sender, WindowEventArgs args)
 			StoreWindowPlacementData();
 		}
 
-		private void WindowEx_Activated(object sender, WindowActivatedEventArgs args)
-		{
-			if (AppLifecycleHelper.IsLaunchInitialized && SystemBackdrop is AppSystemBackdrop appSystemBackdrop)
```

Comment:
Why not just comment out here?

Reply:
Fixing this is low priority and it's best to remove the code if it's not going to be used.

Reply:
Okay, I'll revert this PR when I find a way to fix this.